tests/jit/jit_models_test.py

```diff
@@ -46,14 +48,11 @@ def small_model_params():
             "kernel_size": 32,
             "stride": 16,
         },
-        LSTMTasNet.__name__: {
-            "n_src": 2,
-            "hid_size": 4,
-            "n_layers": 1,
-            "dropout": 0.0,
-            "n_filters": 32,
-            "kernel_size": 32,
-            "stride": 16,
+        DCCRNet.__name__: {
+            "architecture": "DCCRN-CL",
+        },
+        DCUNet.__name__: {
+            "architecture": "DCUNet-10",
```

Comment:
Is it possible to make "mini" architectures for this in their respective files? For tests, demos on collab etc.. That would be cool.

Reply:
Makes sense, I will try

Reply:
I did it. Have a look if you can.